snap-overlay/bin/launch.sh

```diff
@@ -64,4 +66,4 @@ while :; do
     fi
 done
 
-echo "You can also visit the openstack dashboard at 'http://10.20.20.1/'"
+echo "You can also visit the openstack dashboard at 'http://$extgateway/'"
```

Comment:
We have a fun situation where snapcraft wants variables with dashes as separators, and jinja wants variables with underscores. Each will break if you use the other's convention, of course.

I'm doing an end run around the issue by just smooshing all our variable names into one word, but it might be a good idea to write a converter into snap.openstack.